.github/workflows/check.yaml

```diff
@@ -28,6 +35,10 @@ jobs:
       additional-env-vars: |
         _R_CHECK_CRAN_INCOMING_REMOTE_=false
       additional-r-cmd-check-params: --as-cran
+      enforce-note-blocklist: true
+      note-blocklist: '*'
```

Comment:
does that mean that we want to fail on any note? We don't want that. We have identified already a list of notes that we want to fail on -> https://github.com/insightsengineering/idr-tasks/issues/505